.github/workflows/job-precommit-python-filesystem-io.yml

```diff
@@ -0,0 +1,112 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# To learn more about GitHub Actions in Apache Beam check the CI.md
+
+# Runs PreCommit Python Filesystem IO scheduled on cron, on push to master and on pull request
+
+name: PreCommit Python Filesystem IO
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+    tags: 'v*'
+  pull_request_target:
+    branches: ['master', 'release-*']
+    tags: 'v*'
+    paths: [ 'sdks/python/apache_beam/io/filesystemio.py',
+             'sdks/python/apache_beam/io/filesystemio_test.py',
+             'sdks/python/apache_beam/io/filebasedsink.py',
+             'sdks/python/apache_beam/io/filebasedsink_test.py',
+             'sdks/python/apache_beam/io/filebasedsource.py',
+             'sdks/python/apache_beam/io/filebasedsource_test.py',
+             'sdks/python/apache_beam/io/filesystem.py',
+             'sdks/python/apache_beam/io/filesystem_test.py',
+             'sdks/python/apache_beam/io/fileio.py',
+             'sdks/python/apache_beam/io/fileio_test.py',
+             'sdks/python/apache_beam/io/filesystems.py',
+             'sdks/python/apache_beam/io/filesystems_test.py']
```

Comment:
Would we be better off doing something like `sdks/python/apache_beam/io/file*`? I haven't tested that syntax, but it would help us avoid future problems if more file* files are introduced as well